### ISSUE TITLE
Fails on windows when username contains a single quote

### ISSUE THREAD

**Issue by tylerflint** [nanobox-io/nanobox#494]
```
λ nanobox-update

The update process requires Administrator privileges.
Another window will be opened as the Administrator to continue this process.
Enter to continue:

At line:1 char:70
+ ... Process 'cmd' -ArgumentList '/c C: & cd C:\Users\Ben O'Neal & nanobox ...
+                                                                 ~
The ampersand (&) character is not allowed. The & operator is reserved for future use; wrap an
ampersand in double quotation marks ("&") to pass it as part of a string.
At line:1 char:125
+ ... box-update "C:\Program Files\Nanobox\nanobox.exe"' -Verb RunAs -Wait}
+                                                      ~~~~~~~~~~~~~~~~~~~~
The string is missing the terminator: '.
At line:1 char:3
+ & {Start-Process 'cmd' -ArgumentList '/c C: & cd C:\Users\Ben O'Neal  ...
+   ~
Missing closing '}' in statement block or type definition.
    + CategoryInfo          : ParserError: (:) [], ParentContainsErrorRecordException
    + FullyQualifiedErrorId : AmpersandNotAllowed
```

**Comment by tylerflint** [comment]
We'll need to figure out how to properly escape a windows command. This link suggests using `^`. I have yet to try it: https://ss64.com/nt/syntax-esc.html

**Comment by tylerflint** [comment]
Here's a great writeup of the problem and also the solution: https://stackoverflow.com/questions/20958388/command-line-escaping-single-quote-for-powershell

**Comment by lyondhill** [comment]
I believe this is fixed by #493 
reopen the issue if it comes up again
